akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -97,6 +110,45 @@ private[akka] object Running {
   import InternalProtocol._
   import Running.RunningState
 
+  import scala.concurrent.duration._
+  import akka.actor.typed.scaladsl.AskPattern._
+
+  private val log = setup.log
+
+  def startReplicationStream(state: RunningState[S], aa: ActiveActive): Unit = {
+    val query = PersistenceQuery(setup.context.system)
+    aa.allReplicas.zipWithIndex.foreach {
+      case (replica, _) =>
```

Comment:
The index is not being used. I assumed this is a preparation for https://github.com/akka/akka/issues/29257 and we will have another collection to fish for plugin ids using the index, but `allReplicas` is a `Set` so order not guaranteed.

Reply:
this is how it worked in multi dc. I'll remove the index for now as i think we may do it differently and have the user pass in the query plugin id